container.go

```diff
@@ -135,6 +135,8 @@ type NetworkSettings struct {
 	Bridge      string                 `json:"Bridge,omitempty" yaml:"Bridge,omitempty"`
 	PortMapping map[string]PortMapping `json:"PortMapping,omitempty" yaml:"PortMapping,omitempty"`
 	Ports       map[Port][]PortBinding `json:"Ports,omitempty" yaml:"Ports,omitempty"`
+	NetworkID   string                 `json:"NetworkID,omitempty" yaml:"NetworkID,omitempty"`
```

Comment:
It's "NetworkID":

        "MacAddress": "02:42:ac:11:2a:02",
        "NetworkID": "b6408b405133c5475bfde2ad383b80732326e1fb9bc713f6586552836ba2fdd5",
        "PortMapping": null,
        "Ports": {},
        "SandboxKey": "/var/run/docker/netns/23659bc4a779",
        "SecondaryIPAddresses": null,
        "SecondaryIPv6Addresses": null